metadata_service/proxy/atlas_proxy.py

```diff
@@ -348,7 +348,7 @@ def _serialize_columns(self, *, entity: EntityUniqueAttribute) -> \
                     end_epoch = stats_attrs.get('end_epoch')
 
                     statistics.append(
-                        Statistics(
+                        Stat(
```

Comment:
I thought we weren't changing this?

Reply:
Ah, we' added a new `Stat` class that will supercede `Statistics` once everyone has moved over to it. Just doing a little of that cleanup while I'm here!

Reply:
oh great!